autobuild.xml

```diff
@@ -3203,8 +3201,7 @@ Copyright (c) 2012, 2014, 2015, 2016 nghttp2 contributors</string>
                 <key>options</key>
                 <array>
                   <string>-G</string>
-                  <string>Ninja</string>
-                  <string>-DUSE_OPENAL:BOOL=ON</string>
+                  <string>Unix Makefiles</string>
```

Comment:
We should stick to one build system, either ninja (as it is still in the ReleaseOS config) or Makefiles.

ninja used to be faster, but I haven't tested that in a while. Generally I do not mind either, just that we use *one* standard and not multiple

Reply:
I've unified both configurations to the `Unix Makefiles` generator.